Example/NYTPhotoViewer-SwiftTests/NYTPhotosOverlayViewTests.swift

```diff
@@ -20,21 +20,21 @@ class NYTPhotosOverlayViewTests: XCTestCase {
         let overlayView = NYTPhotosOverlayView()
         let leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: UIBarButtonSystemItem.Done, target: nil, action: nil)
         overlayView.leftBarButtonItem = leftBarButtonItem
-        XCTAssert(leftBarButtonItem == overlayView.navigationBar.items.first?.leftBarButtonItem)
+        XCTAssert(leftBarButtonItem == overlayView.navigationBar.items!.first?.leftBarButtonItem)
```

Comment:
My thinking here was, it's a test, if it crashes so be it, that's still a failure. But maybe we ought to just let `nil` come through and have the assertion fail. Will send an updated PR.